glustercli/cmd/snapshot-delete.go

```diff
@@ -8,8 +8,10 @@ import (
 )
 
 const (
-	snapshotDeleteHelpShort = "Delete a Gluster Snapshot"
-	snapshotDeleteHelpLong  = "Delete a Gluster snapshot of the given name."
+	snapshotDeleteHelpShort    = "Delete snapshots"
+	snapshotDeleteHelpLong     = "It delete a snapshot if a snapshot name is given as an argument. It also takes command all to delete all snapshots in a cluster or all snapshots of a particular volume"
```

Comment:
`It delete a snapshot` to `It deletes a snapshot`

Reply:
done